libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -154,6 +154,15 @@ const AP_Param::GroupInfo AP_Landing::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("FLARE_AIM", 17, AP_Landing, flare_effectivness_pct, 50),
 
+    // @Param: WIND_COMP
+    // @DisplayName: Headwind Compensation when Landing
+    // @Description: This param controls how much headwind compensation is used when landing.  Headwind speed component multiplied by this parameter is added to TECS_LAND_ARSPD command.  Set to Zero to disable.  Note:  The target landing airspeed command is still limited to ARSPD_FBW_MAX.
+    // @Range: 0 1
+    // @Units: %
```

Comment:
units are wrong, either make it 50%, or as a proportion

Reply:
converted to percent

Reply:
Percent range is 0 to 100.